src/config/mongoose_config_parser_toml.erl

```diff
@@ -471,30 +472,700 @@ process_module([Mod|_] = Path, Opts) ->
+    [{all_can_invite, V}];
+module_opt([<<"max_occupants">>, <<"mod_muc_light">>|_], <<"infinity">>) ->
+    [{max_occupants, infinity}];
+module_opt([<<"max_occupants">>, <<"mod_muc_light">>|_], V) ->
+    [{max_occupants, V}];
+module_opt([<<"rooms_per_page">>, <<"mod_muc_light">>|_], <<"infinity">>) ->
+    [{rooms_per_page, infinity}];
+module_opt([<<"rooms_per_page">>, <<"mod_muc_light">>|_], V) ->
+    [{rooms_per_page, V}];
+module_opt([<<"rooms_in_rosters">>, <<"mod_muc_light">>|_], V) ->
+    [{rooms_in_rosters, V}];
+module_opt([<<"config_schema">>, <<"mod_muc_light">>|_] = Path, V) ->
+    Configs = parse_list(Path, V),
+    [{config_schema, Configs}];
 module_opt([<<"access_max_user_messages">>, <<"mod_offline">>|_], V) ->
     [{access_max_user_messages, b2a(V)}];
+module_opt([<<"send_pings">>, <<"mod_ping">>|_], V) ->
+    [{send_pings, V}];
+module_opt([<<"ping_interval">>, <<"mod_ping">>|_], V) ->
+    [{ping_interval, V}];
+module_opt([<<"timeout_action">>, <<"mod_ping">>|_], V) ->
+    [{timeout_action, b2a(V)}];
+module_opt([<<"ping_req_timeout">>, <<"mod_ping">>|_], V) ->
+    [{ping_req_timeout, V}];
+module_opt([<<"host">>, <<"mod_pubsub">>|_], V) ->
+    [{host, b2l(V)}];
+module_opt([<<"access_createnode">>, <<"mod_pubsub">>|_], V) ->
+    [{access_createnode, b2a(V)}];
+module_opt([<<"max_items_node">>, <<"mod_pubsub">>|_], V) ->
+    [{max_items_node, V}];
+module_opt([<<"max_subscriptions_node">>, <<"mod_pubsub">>|_], V) ->
+    [{max_subscriptions_node, V}];
+module_opt([<<"nodetree">>, <<"mod_pubsub">>|_], V) ->
+    [{nodetree, V}];
+module_opt([<<"ignore_pep_from_offline">>, <<"mod_pubsub">>|_], V) ->
+    [{ignore_pep_from_offline, V}];
+module_opt([<<"last_item_cache">>, <<"mod_pubsub">>|_], V) ->
+    [{last_item_cache, b2a(V)}];
+module_opt([<<"plugins">>, <<"mod_pubsub">>|_] = Path, V) ->
+    Plugs = parse_list(Path, V),
+    [{plugins, Plugs}];
+module_opt([<<"pep_mapping">>, <<"mod_pubsub">>|_] = Path, V) ->
+    Mappings = parse_list(Path, V),
+    [{pep_mapping, Mappings}];
+module_opt([<<"default_node_config">>, <<"mod_pubsub">>|_] = Path, V) ->
+    Config = parse_list(Path, V),
+    [{default_node_config, Config}];
+module_opt([<<"item_publisher">>, <<"mod_pubsub">>|_], V) ->
+    [{item_publisher, V}];
+module_opt([<<"sync_broadcast">>, <<"mod_pubsub">>|_], V) ->
+    [{sync_broadcast, V}];
+module_opt([<<"pool_name">>, <<"mod_push_service_mongoosepush">>|_], V) ->
+    [{pool_name, b2a(V)}];
+module_opt([<<"api_version">>, <<"mod_push_service_mongoosepush">>|_], V) ->
+    [{api_version, b2l(V)}];
+module_opt([<<"max_http_connections">>, <<"mod_push_service_mongoosepush">>|_], V) ->
+    [{max_http_connections, V}];
+module_opt([<<"access">>, <<"mod_register">>|_], V) ->
+    [{access, b2a(V)}];
 module_opt([<<"welcome_message">>, <<"mod_register">>|_], V) ->
     [{welcome_message, {b2l(V)}}];
+module_opt([<<"registration_watchers">>, <<"mod_register">>|_], V) ->
+    [{registration_watchers, V}];
```

Comment:
This seems to be a list of binaries, I thing we should call `parse_list` here.

Reply:
The handler would just pass the value in that case so I though it was not necessary, will change that